sdk/clientcore/core/pom.xml

```diff
@@ -181,6 +183,32 @@
           </dependency>
         </dependencies>
       </plugin>
+
+      <plugin>
+        <groupId>com.azure.tools</groupId>
+        <artifactId>codesnippet-maven-plugin</artifactId>
+        <version>1.0.0-beta.10</version> <!-- {x-version-update;com.azure.tools:codesnippet-maven-plugin;external_dependency} -->
+        <configuration>
+          <additionalCodesnippets>
+            <additionalCodesnippet>
+              <root>src/test</root>
+              <glob>**/*.java</glob>
+            </additionalCodesnippet>
+          </additionalCodesnippets>
+        </configuration>
+      </plugin>
```

Comment:
Should this plugin be in the parent pom?

Reply:
The plugin is in the parent POM, this is extending the configurations to pull snippets from test code. Or, are you saying we should put the configuration to add test code as snippet sources to the parent POM?

Reply:
Yeah, we should make this the default to allow test code to be snippet sources.